libs/perun/services/src/lib/auth.service.ts

```diff
@@ -72,7 +82,7 @@ export class AuthService {
       scope: this.store.get('oidc_client', 'oauth_scopes') as string,
       clockSkewInSec: clockSkewInSec,
       timeoutFactor: timeoutFactor,
-      // sessionChecksEnabled: true,
+      userinfoEndpoint: this.store.get('oidc_client', 'user_info_endpoint_url') as string,
```

Comment:
Add to `OidcClient` if needed.
```suggestion
      userinfoEndpoint: this.store.getProperty('oidc_client').user_info_endpoint_url,
```